packages/modelviewer.dev/index.html

```diff
@@ -85,8 +85,10 @@ <h3 class="grouping-title border-bottom">Examples</h3>
             <li><a href="examples/customizing-ui.html"><h4>Customizing UI</h4></a></li>
 
             <li><a href="examples/scene-graph.html"><h4>Scene Graph</h4></a></li>
-			
-			      <li><a href="examples/tester.html"><h4>Interactive Example</h4></a></li>
+
+            <li><a href="examples/tester.html"><h4>Interactive Example</h4></a></li>
+
+            <li><a href="./node_modules/@google/model-viewer-space-opera/index.html"><h4>Space Opera (Editor)</h4></a></li>
```

Comment:
I don't think we need to publicize the Space Opera name quite this widely :). Maybe just call it Model Editor?

Reply:
Done